### PR TITLE
Chewbbaca

### PR DESCRIPTION
**Pull request by nilchia**

### CODE REVIEW

File: files/galaxy/tpv/tools.yml [hunk 1]
```diff
@@ -303,6 +303,8 @@ tools:
       require:
         - docker
 
+  toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca_createschema/.*:
```
Comment (bgruening) [inline files/galaxy/tpv/tools.yml:306]: what about the other chewbbaca tools?

Comment (nilchia) [reply files/galaxy/tpv/tools.yml:306]: If I add `toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca*` does that work for all chewbbaca packages?

Comment (bgruening) [reply files/galaxy/tpv/tools.yml:306]: I think so. @sanjaysrikakulam ?

File: files/galaxy/tpv/tools.yml [hunk 2]
```diff
@@ -303,6 +303,8 @@ tools:
       require:
         - docker
 
+  toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca*:
```
Comment (sanjaysrikakulam) [inline files/galaxy/tpv/tools.yml:306]:
```suggestion
  toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca.*:
```

File: files/galaxy/tpv/tools.yml (hunk 1 above)
Comment (sanjaysrikakulam) [reply files/galaxy/tpv/tools.yml]: We can give it a shot. To my understanding this `toolshed.g2.bx.psu.edu/repos/iuc/chewbbaca.*` should work. I have made a suggestion to add a `.` to the regex.

Comment (nilchia) [reply files/galaxy/tpv/tools.yml]: Thanks @sanjaysrikakulam

**Review by bgruening — APPROVED** [review]

**Pull request merged**